src/components/MDXComponents.tsx

```diff
@@ -43,6 +43,7 @@ const MDXComponents = {
       <NextLink href={href} passHref>
         <Link
           isExternal={href.startsWith('http') && !href.includes('geth.ethereum.org')}
+          // TODO: move color to theme
           color='#18bc9c'
```

Comment:
```suggestion
          color='brand.light.primary'
```

I dont see this color in the design system, but I think this is what this should be

Reply:
Or we come back to this when doing the docs. Feel free to ignore this if that is the idea

Reply:
@corwintines yeah I think I grabbed that color from the figma design (not design system) so it's not defined yet, think it could be added later